src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.12.mdx

```diff
@@ -0,0 +1,21 @@
+---
+subject: "Lambda-Extension"
+releaseDate: '2024-06-24'
+version: 2.3.12
+---
+
+
+## Notes
+
+* Adds local testing for Extension. Local testing is enabled by setting `NEWRELIC_LOCAL_TESTING` to true. Currently it supports only Python runtime. It generates the lambda layers for `x86` and `arm` architecture layers using `publish.sh` script and invokes the test Lambdas using `test.sh` script.
```

Comment:
Can we please add back ticks to true, example `true`

Reply:
Suggestion for rephrase:
Adds local testing for Extension. Local testing is enabled by setting `NEWRELIC_LOCAL_TESTING` to true. Currently supports Python runtime only, and generates lambda layers for `x86` and `arm` architecture using `publish.sh` and invokes the test Lambdas using `test.sh` script.